.github/workflows/identify-old-issues-and-pr.yml

```diff
@@ -0,0 +1,29 @@
+name: Identify old issues and PR
```

Comment:
I don't think that we still need the python script. I found a stale github actions, but I'd prefer not implementing it yet. There are issues I don't want closed. What would you recommend?

Reply:
Yup, We can migrate the python scripts in the next round. I prefer one at a time.

Reply:
This shall be tackled in a different PR